app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -610,10 +610,20 @@ public boolean onSupportNavigateUp() {
 
     @Override
     public void onBackPressed() {
-        super.onBackPressed();
-        if (isEditMode)
-            AnalyticsUtil.cancelEditContent(this);
-        else
-            AnalyticsUtil.cancelAddContent(this);
+        new AlertDialog.Builder(MemoryEditActivity.this)
+                .setTitle("Discard Memory")
```

Comment:
This is not using string resources. Please use them and also do the Dutch translation.